x-pack/plugins/security_solution/public/management/pages/response_actions/view/response_actions_list.tsx

```diff
@@ -0,0 +1,422 @@
+          },
+        ];
+
+        itemIdToExpandedRowMapValues[item.id] = (
+          <EuiFlexGroup>
+            {[descriptionListLeft, descriptionListCenter, descriptionListRight].map((list) => (
+              <EuiFlexItem>
+                <EuiDescriptionList listItems={list} compressed={true} />
+              </EuiFlexItem>
+            ))}
+          </EuiFlexGroup>
+        );
+      }
+      setItemIdToExpandedRowMap(itemIdToExpandedRowMapValues);
+    },
+    [itemIdToExpandedRowMap]
+  );
+
+  // table column
+  const responseActionListColumns = useMemo(() => {
+    const columns = [
+      {
+        field: 'startedAt',
+        name: TABLE_COLUMN_NAMES.time,
+        width: '15%',
+        truncateText: true,
+        render: (startedAt: ActionDetails['startedAt']) => {
+          return (
+            <FormattedDate
+              fieldName={TABLE_COLUMN_NAMES.time}
+              value={startedAt}
+              className="eui-textTruncate"
+            />
+          );
+        },
+      },
+      {
+        field: 'command',
+        name: TABLE_COLUMN_NAMES.command,
+        width: '10%',
+        truncateText: true,
+        render: (command: ActionDetails['command']) => {
+          return (
+            <EuiToolTip content={command} anchorClassName="eui-textTruncate">
+              <FormattedMessage
+                id="xpack.securitySolution.responseActionsList.list.item.command"
+                defaultMessage="{command}"
+                values={{ command }}
+              />
+            </EuiToolTip>
+          );
+        },
+      },
+      {
+        field: 'logEntries',
+        name: TABLE_COLUMN_NAMES.user,
+        width: '14%',
+        truncateText: true,
+        render: (logEntries: ActionDetails['logEntries']) => {
+          const userId =
+            (logEntries.filter((e) => e.type === 'action')[0] as EndpointActivityLogAction).item
+              .data.user.id || '-';
+          return (
+            <StyledFacetButton
+              icon={
+                <EuiAvatar
+                  name={userId}
+                  data-test-subj={getTestId('column-user-avatar')}
+                  size="s"
+                />
+              }
+            >
+              <EuiToolTip content={userId} anchorClassName="eui-textTruncate">
+                <EuiText
+                  size="s"
+                  className="eui-textTruncate eui-fullWidth"
+                  data-test-subj={getTestId('column-user-name')}
+                >
+                  <p className="eui-displayInline eui-TextTruncate">{userId}</p>
+                </EuiText>
+              </EuiToolTip>
+            </StyledFacetButton>
+          );
+        },
+      },
+      {
+        field: 'agents',
+        name: TABLE_COLUMN_NAMES.host,
+        width: '20%',
+        truncateText: true,
+        render: (agents: ActionDetails['agents']) => {
+          // TODO: compute host names later with hostMetadata? (using agent Ids for now)
```

Comment:
Neat idea! I'll see if I can do this in the same PR too

Reply:
Hostnames only is not very useful. Most of the time you likely need the ID as well and probably other data from the host `metadata` document.

You don't have to include this in your PR. it can be a separate/future PR/task.